app/controllers/api/books_controller.rb

```diff
@@ -8,5 +8,9 @@ def index
       end)
     end
 
+    def show
+      render json: { book: Book.find(params[:id]) }
+    end
+
```

Comment:
@resivalex I am very surprised that rubocop doesn't complain here. It should tell `Layout/TrailingWhitespace: Trailing whitespace detected.`. Do you know why is it so?

Reply:
```
Layout/EmptyLinesAroundClassBody:
  EnforcedStyle: empty_lines_except_namespace
```
Look at this one, please

Reply:
Gods what's going on

Reply:
@resivalex ah I see thanks. I think that default option (`no_empty_lines`) produces a bit more cleaner code but it's just a personal taste. Let's leave it like that then for now.

Reply:
It's you lucky day @isabloCh haha!

Reply:
🤣